Tapawingo_backend/Controllers/RoutesController.cs

```diff
@@ -24,5 +27,22 @@ public IActionResult GetRoute(int id)
                 Ok(organisation) :
                 NotFound("Organisation with this id was not found.");
         }
+
+        [HttpPost]
+        [ProducesResponseType(StatusCodes.Status201Created, Type = typeof(RouteDto))]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult CreateRoute([FromBody] CreateRouteDto createRouteDto)
+        {
+            var newRoute = _routesService.CreateRoute(createRouteDto);
+            if (newRoute != null)
```

Comment:
We maken op een andere plek gebruik van een inline if-else, genaamd een 'ternary conditional operator'. Als je niet weet hoe dit werkt, zie: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator

Kunnen beter consistent dezelfde manier toepassen.